src/main.rs

```diff
@@ -25,7 +26,42 @@ fn main() -> Result<(), eframe::Error> {
         ..Default::default()
     };
 
-    eframe::run_native("Contrast", options, Box::new(|_cc| Box::new(MyApp::new())))
+    let path: Option<PathBuf> = match env::args().nth(1) {
+        Some(relative_path) => {
+            let path = PathBuf::from(relative_path);
+            match fs::canonicalize(path) {
+                Ok(p) => Some(PathBuf::from(
+                    p.to_str()
+                        .unwrap()
+                        .to_string()
+                        // windows, am i rite?
+                        .strip_prefix(&"\\\\?\\".to_string())
+                        .unwrap(),
+                )),
+                Err(err) => {
+                    eprintln!("Invalid path: {err}");
+                    None
+                }
+            }
+        }
+        None => None,
+    };
```

Comment:
This should not error if the path is invalid. Instead transform the result into an Option and keep working with that. If there is no valid path provided, the app should start without a selected project.

Reply:
> If there is no valid path provided, the app should start without a selected project.
This is the current behavior. It just prints to command line to give some feedback to the user.

Reply:
The calls to .unwrap() should end execution I am pretty sure.